stdlib/string.gr

```diff
@@ -1297,3 +1297,107 @@ export let decode = (bytes: Bytes, encoding: Encoding) => {
 export let decodeKeepBom = (bytes: Bytes, encoding: Encoding) => {
   decodeHelp(bytes, encoding, false)
 }
+
+/**
+ * Iterates over Unicode code points in a string.
+ * 
+ * @param f: The iterator function
+ * @param s: The string to iterate
+ */
+@disableGC
+export let forEachCodePoint = (f: (Number) -> Void, s: String) => {
```

Comment:
```suggestion
 * @param fn: The iterator function
 * @param str: The string to iterate
 */
@disableGC
export let forEachCodePoint = (fn: (Number) -> Void, str: String) => {
```
I have this nit about single character variables (and I want the `@param` to match). This would need to be updated in the function body too.